programs/nodeos/main.cpp

```diff
@@ -147,6 +147,9 @@ enum return_codes {
 
 int main(int argc, char** argv)
 {
+
+   ilog("nodeos started");
```

Comment:
why this log? if we do, probably should be "nodeos starting".

Reply:
Can be safely removed